### ISSUE TITLE
aria-expanded bugs on flyout demo

### ISSUE THREAD

**Issue by goetsu** [w3c/wai-tutorials#303]
Hi, 
I think there is two bugs with http://www.w3.org/WAI/tutorials/menus/flyout/#improve-screen-reader-support-using-wai-aria demo : 
- when the submenu is active and I navigate through the submenu, the focus go to "MarsCars", the submenu is closed but without changing the aria-expanded value to false
- as "SpaceBears" is not a link but a button at least it must have the role="button" and must be activated with enter or spacebar

**Comment by yatil** [comment]
Thanks for the comment, I’ll have a look.

**Comment by goetsu** [comment]
there is also some strange aria-presentation="true" on the nav element containing the menu

**Comment by yatil** [comment]
The aria-presentation="true" on the nav element is there so the nav element is not announced by assistive technologies when navigating the page as this is a demo example and having a navigation role that doesn’t belong to the website is obviously very strange.

**Comment by goetsu** [comment]
so I think it must be role="presentation" not aria-presentation and in that case you have to remove the role="navigation" too

**Comment by yatil** [comment]
Ah, I see you’re right. Will fix that asap. How did I come up with aria-presentation?! :-D
